spec/src/test/java/impl/UserQuestionResourceTest.java

```diff
@@ -135,20 +118,20 @@ public void shouldBePossibleToUpdateQuestion() {
 
         Auth     mockAuth = new MockAuth(createdUser.getId());
         mockAuth.setUserId(createdUser.getId());
-        questionResource.postQuestion(mockAuth, question).toBlocking().singleOrDefault(null);
+        questionResource.createQuestion(mockAuth, question).toBlocking().singleOrDefault(null);
 
-        long newQuestionId = userQuestionResource.getQuestions(createdUser.getId(), new CollectionOptions()).toBlocking().single().get(0).getId();
+        long newQuestionId = userQuestionResource.getQuestions(createdUser.getId()).toBlocking().single().get(0).getId();
         question.setBounty(400);
         question.setTitle("new title");
         question.setQuestion("new question body");
         userQuestionResource.updateQuestion(mockAuth, newQuestionId, question).toBlocking().singleOrDefault(null);
-        List<Question> questions = userQuestionResource.getQuestions(createdUser.getId(), new CollectionOptions()).toBlocking().single();
+        List<Question> questions = userQuestionResource.getQuestions(createdUser.getId()).toBlocking().single();
         assertEquals(1, questions.size());
 
         Question updatedQuestion = questions.get(0);
         assertEquals("new title", updatedQuestion.getTitle());
         assertEquals("new question body", updatedQuestion.getQuestion());
-        assertEquals(new Integer(400),updatedQuestion.getBounty());
+        assertEquals(new Integer(300),updatedQuestion.getBounty());
```

Comment:
i changed so that only title and question can be modified through the api.

Reply:
It's kinda magical here since it set by TestSetup. We should either add bounty as a param, or use a new Question object for the update and assert using the value from the "original" object.